src/applications/ezr/containers/App.jsx

```diff
@@ -41,7 +61,7 @@ const App = props => {
       }
       // eslint-disable-next-line react-hooks/exhaustive-deps
     },
-    [isUserLOA3],
+    [fetchEnrollmentStatus, isUserLOA3],
```

Comment:
Lint was complaining about this not being present.

Reply:
The linting exemption comment on L62 needs moved to L63. I would not make that fetch function a dependency.